src/test/java/seedu/address/logic/LogicManagerTest.java

```diff
@@ -80,8 +81,11 @@ public void execute_storageThrowsIoException_throwsCommandException() {
 
         // Execute add command
         String addCommand = AddCommand.COMMAND_WORD + NAME_DESC_AMY + PHONE_DESC_AMY + EMAIL_DESC_AMY
-                + ADDRESS_DESC_AMY;
-        Person expectedPerson = new PersonBuilder(AMY).withTags().build();
+                + ADDRESS_DESC_AMY + TELEGRAM_DESC_AMY;
+        Person expectedPerson = new PersonBuilder(AMY).withGroupTags().build();
+        System.out.println("HIIIIII");
+        System.out.println(addCommand);
+        System.out.println(expectedPerson);
```

Comment:
Print statements should be removed.

Reply:
Oh Whoops! Nice catch. That was embarassing!